NDTensors/test/device_list.jl

```diff
@@ -12,7 +13,7 @@ function devices_list(test_args)
   end
 
   if "cuda" in test_args || "all" in test_args
-    CUDA.allowscalar()
+    #CUDA.allowscalar()
```

Comment:
Reminder to delete this once we switch to using `@allowscalar`.

Reply:
I have removed this line, thanks!